### PR TITLE
Add translation package

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

**Comment by xepozz** [comment]
image
image
Added two languages supporting for some texts in main layout.

File: config/routes.php [hunk 1]
```diff
@@ -32,6 +32,9 @@
     Route::get('/')
         ->action([SiteController::class, 'index'])
         ->name('site/index'),
+    Route::get('/locale')
```
Comment (samdark) [inline config/routes.php:35]: Using GET to change application state isn't a very good idea.

Comment (xepozz) [reply config/routes.php:35]: It's just for shortcut purpose while developing.

File: views/layout/main.php [hunk 2]
```diff
@@ -2,19 +2,14 @@
...
 declare(strict_types=1);
 
-use App\Asset\AppAsset;
-use App\Widget\PerformanceMetrics;
-use Yiisoft\Form\Widget\Form;
-use Yiisoft\Html\Html;
-use Yiisoft\Strings\StringHelper;
-use Yiisoft\Yii\Bootstrap5\Nav;
-use Yiisoft\Yii\Bootstrap5\NavBar;
+use App\Asset\AppAsset;use App\Widget\PerformanceMetrics;use Yiisoft\Form\Widget\Form;use Yiisoft\Html\Html;use Yiisoft\Strings\StringHelper;use Yiisoft\Yii\Bootstrap5\ButtonDropdown;use Yiisoft\Yii\Bootstrap5\Nav;use Yiisoft\Yii\Bootstrap5\NavBar;
```
Comment (rustamwin) [inline views/layout/main.php:5]: Wrong format :)

File: src/Command/Translation/TranslateCommand.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+namespace App\Command\Translation;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Yiisoft\Translator\TranslatorInterface;
+
+final class TranslateCommand extends Command
```
Comment (samdark) [inline src/Command/Translation/TranslateCommand.php:13]: What's the purpose of this command?

Comment (xepozz) [reply src/Command/Translation/TranslateCommand.php:13]: It's only for debugging purpose.

**Comment by samdark** [comment]
👍

**Comment by rossaddison** [comment]
Clicking on 'Russian' instead of 'English' does not change the text on screen to Russian. The cookie 'locale' is not being set since the cookie collection only contains the PHPSESSID in a var_dump. Any suggestions?

**Pull request merged**